sumologic/sumologic_cse_rule_common.go

```diff
@@ -71,7 +71,9 @@ func resourceToStringArray(resourceStrings []interface{}) []string {
 	result := make([]string, len(resourceStrings))
 
 	for i, resourceString := range resourceStrings {
-		result[i] = resourceString.(string)
+		if resourceString != nil {
+			result[i] = resourceString.(string)
+		}
```

Comment:
tried that, but it complains when checks/validates the resource ( tags mismatch )
added a ValidateFunc: validation.StringIsNotEmpty on tags elements instead to avoid user passing empty tags.